### ISSUE TITLE
Suggestion should include actual substitution used

### ISSUE THREAD

**Issue by singpolyma** [dropbox/zxcvbn#128]
If I use "hel3nhunt" it says "Predictable substitutions like '@' instead of 'a' don't help very much" -- why not just say "like '3' instead of 'e'" ?

**Comment by KunalTanwar** [comment]
Good point — fixed in [zxcvbn-ts](https://github.com/KunalTanwar/zxcvbn-ts)! The l33t suggestion now reads the actual substitution from the match's sub_display field and uses it directly, so instead of always saying "like '@' instead of 'a'" it will say something like "like '3 -> e'" based on what was actually used.
